Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -505,6 +509,39 @@ final class PackageToolTests: XCTestCase {
                 ["CustomNameTests"])
         }
     }
+    
+    func testCreate() throws {
+        try XCTSkipIf(InitPackage.createPackageMode == .legacy)
+        try testWithTemporaryDirectory { tmpPath in
+            let fs = localFileSystem
+            let path = tmpPath.appending(component: "MyExe")
+            
+            _ = try execute(["create", "MyExe"], packagePath: tmpPath)
+            
+            XCTAssert(fs.exists(path.appending(component: "Package.swift")))
+            XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Sources")), ["main.swift"])
+        }
+    }
+    
+    func testCreateLibrary() throws {
+        try XCTSkipIf(InitPackage.createPackageMode == .legacy)
+        try testWithTemporaryDirectory { tmpPath in
+            let fs = localFileSystem
+            let path = tmpPath.appending(component: "MyLib")
+            
+            _ = try execute(["create", "MyLib", "--type", "library"], packagePath: tmpPath)
+            
+            XCTAssert(fs.exists(path.appending(component: "Package.swift")))
+            XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Sources")), ["MyLib.swift"])
+        }
+    }
+    
+    func testCreateNoNameArgument() throws {
+        try XCTSkipIf(InitPackage.createPackageMode == .legacy)
+        let result = try SwiftPMProduct.SwiftPackage.executeProcess(["create"])
+        let stderrOutput = try result.utf8stderrOutput()
+        XCTAssert(stderrOutput.contains("error: Missing expected argument '<package-name>'"), #"actual: "\#(stderrOutput)""#)
+    }
```

Comment:
are we missing tests for crating and initializing from a template?

Reply:
Yes, tests are still needed.